src/mousetracker.js

```diff
@@ -3248,5 +3264,33 @@
             } );
         }
     }
+    
+    /**
+     * @function
+     * @private
+     * @inner
+     * @returns {Boolean} True if inside an iframe, otherwise false.
+     */
+    function isInIframe () {
+        try {
+            return window.self !== window.top;
+        } catch (e) {
+            return true;
+        }
+    }
```

Comment:
I am wondering if this could be cached in a variable like this:
```javascript
var isInFrame = (function() {
    try {
        return window.self !== window.top;
    } catch (e) {
        return true;
    }
})();
```